ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/traffic-policies.md

```diff
@@ -0,0 +1,92 @@
+# traffic-policies
+# Table of Contents
+<!-- toc -->
+
+- [Management](#management)
+  - [Management Interfaces](#management-interfaces)
+- [Authentication](#authentication)
+- [Monitoring](#monitoring)
+- [Internal VLAN Allocation Policy](#internal-vlan-allocation-policy)
+  - [Internal VLAN Allocation Policy Summary](#internal-vlan-allocation-policy-summary)
+- [Interfaces](#interfaces)
+- [Routing](#routing)
+  - [IP Routing](#ip-routing)
+  - [IPv6 Routing](#ipv6-routing)
+- [Multicast](#multicast)
+- [Filters](#filters)
+- [ACL](#acl)
+- [Quality Of Service](#quality-of-service)
+
+<!-- toc -->
+# Management
+
+## Management Interfaces
```

Comment:
there is only one L3 header under L2 header "Management". Can we remove one?
It's cosmetic though.

Reply:
Should be part of another PR related to documentation build as it is not related to traffic policy?